### DISCUSSION TITLE
Mute audio?

### DISCUSSION THREAD

**Discussion by jaredcat** [getgauge/taiko#1938] [General]
Is there a way to mute the chromium browser entirely?  
I already tried ` args: ['−−mute−audio']` and it didn't work

Update: looks like `--disable-audio-output` did the trick

**Comment by sriv** [comment]
strange, I am on linux, and I can see `openBrowser({args: ['--mute-audio']})` working, i.e. it disables audio

**Reply by jaredcat** [reply]
Yea I thought that would be it too, but didn't work on 2 different Windows PCs. Either way looks like one of those will work.
